src/atoms/image/image.tsx

```diff
@@ -58,10 +58,10 @@ export const BaseImage = ({
   return lightbox ? (
     <Fragment>
       <LightboxGallery />
-      <amp-img {...imgAttrs} lightbox={lightbox} />
+      <amp-img {...imgAttrs} lightbox={lightbox} object-fit="contain" />
```

Comment:
This is not an attribute. Can be added via CSS. Refer https://amp.dev/documentation/components/amp-img#styling